src/pages/Lend/Filter/FilterMenuSection.vue

```diff
@@ -49,13 +51,29 @@ export default {
 
 	.filter-items-container {
 		display: none;
-		background-color: #F6FBF6;
+
+		ul {
+			list-style-type: none;
+			margin-bottom: 0;
+		}
+
+		.ais-HierarchicalMenu {
+			a {
+				color: #484848;
+				text-decoration: none;
+			}
+
+			.ais-HierarchicalMenu-item--selected {
+				> a {
+					font-weight: 600;
+					color: $green;
+				}
+			}
+		}
 	}
 
 	&.open {
-		.filter-summary {
-			display: none;
-		}
+		background-color: #F6FBF6;
```

Comment:
Is this that new light green or just another super light gray which we have 3 variations of already? Prob should make it a var in _settings.scss too..

Reply:
It's `$green` with 5% opacity

Reply:
Maybe see if you can do rgba($green, 0.5) I think that's built in...

Reply:
I love that!

Reply:
@foochris @BoulderBrains this is cool